ui/src/app/pages/workspace/workspace-list.spec.tsx

```diff
@@ -1,14 +1,15 @@
-import {mount} from 'enzyme';
+import { mount, ReactWrapper} from 'enzyme';
```

Comment:
Nit: Seems likely this whitespace added after the opening brace was unintended.

Reply:
fixed - this is definitely something I want to cover with the linter soon